plugins/modules/vmware_vm_shell.py

```diff
@@ -47,6 +47,7 @@
       - '   folder: /folder1/datacenter1/vm'
       - '   folder: folder1/datacenter1/vm'
       - '   folder: /folder1/datacenter1/vm/folder2'
+      - The 'vm_id_type' variable is required to be set to 'inventory_path', otherwise 'folder' will be None
```

Comment:
Since we've started using [semantic markup within module documentation](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation), you should rephrase this somehow and use `O(vm_id_type=inventory_path)` instead of `'vm_id_type' variable is required to be set to 'inventory_path'`.

Reply:
ready